client/spec/helpers/desks.js

```diff
@@ -51,6 +51,19 @@ function Desks() {
     };
 
     /**
+     * Returns the stage count for named desk on desks settings list
+     * @param {string} name of desk
+     * @return {string} stage count
```

Comment:
This doesn't seem right, since the method actually returns a _promise_ which is resolved with the stage count?